### PR TITLE
#40: New settings "SCRUBBER_MAPPER" for mapping scrubbers to models

### LINKED ISSUE #40: Question: How to scrub django base user?
Hi there!

Is there a way to define a scrubbing meta class for the base user without deriving from it and creating a new class?

Thx!

### PR DESCRIPTION
**Pull request by GitRon**
closes #40 

@costela What do you say?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,19 @@ Set an alternative locale for Faker used during the scrubbing process.
...
+### `SCRUBBER_MAPPING`:
+
+Define a class and a mapper which does not have to live inside the given model. Useful, if you have no control over the
+models code you'd like to scrub.
+
+````python
+SCRUBBER_MAPPING = {
+    "auth.User": "my_app.scrubbers.UserScrubbers",
+}
+````
```
Comment (costela) [inline README.md:201]: Maybe add a small example of a "pure scrubber" here, similar to #41? (maybe a bit shorter)

**Review by costela — APPROVED** [review]
Oh yes, this feels a lot simpler than #41. Thanks!

I'll let @lociii give a final blessing, but LGTM!

**Comment by GitRon** [comment]
@costela done. Can you estimate when the new version will be out? I actually need this functionality 😅

**Comment by lociii** [comment]
Looks good. Thanks for the contribution (again).

**Comment by lociii** [comment]
I've set myself a reminder for Monday to trigger a new release.

**Comment by GitRon** [comment]
@lociii Awesome! Just give me a short heads-up so I know when I can integrate it in my project 👍

**Comment by regiohelden-dev** [comment]
@GitRon https://pypi.org/project/django-scrubber/0.9.0/ has just been released. Thanks a lot for your effort!

**Pull request merged**